retriever/lib/datapackage.py

```diff
@@ -54,16 +54,16 @@ def get_replace_columns(dialect):
 
 
 def get_nulls(dialect):
-    """Get list of strings that denote null in the dataset"""
-    val = clean_input("nulls (separated by ';') (press return to skip): ",
+    """Get list of strings that denote missing value in the dataset"""
+    val = clean_input("missing_values (separated by ';') (press return to skip): ",
                       split_char=';', ignore_empty=True)
     if val == "" or val == []:
         # return and dont add key to dialect dict if empty val
         return
-    dialect['nulls'] = val
+    dialect['missing_values'] = val
     # change list to single value if size == 1
-    if len(dialect['nulls']) == 1:
-        dialect['nulls'] = dialect['nulls'][0]
+    if len(dialect['missing_values']) == 1:
+        dialect['missing_values'] = dialect['missing_values'][0]
```

Comment:
I'm guessing that these 3-4 lines are what ends up setting the values to be written to the JSON (@henrykironde - can you confirm). Is so then these should all be set to `missingValues`.